sklearn/semi_supervised/label_propagation.py

```diff
@@ -239,19 +240,22 @@ def fit(self, X, y):
 
         n_samples, n_classes = len(y), len(classes)
 
+        alpha = self.alpha
+        if alpha is None:
+            alpha = 0
         y = np.asarray(y)
         unlabeled = y == -1
         clamp_weights = np.ones((n_samples, 1))
-        clamp_weights[unlabeled, 0] = self.alpha
+        clamp_weights[~unlabeled, 0] = alpha
```

Comment:
So Zhou et al suggests this should just be clamp_weights = alpha. Wdyt, @MechCoder ?